From 8bc52dbd9dffb1b2fa4a6aeed2d298d036b619b2 Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Fri, 15 Aug 2014 22:13:06 +0200
Subject: [PATCH 07/37] vfwcap: Drop fallback VfW defines

The defines were added long ago when MinGW still lacked them.
---
 libavdevice/vfwcap.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/libavdevice/vfwcap.c b/libavdevice/vfwcap.c
index 5fdda4a..c067be3 100644
--- a/libavdevice/vfwcap.c
+++ b/libavdevice/vfwcap.c
@@ -27,14 +27,6 @@
 #include <windows.h>
 #include <vfw.h>
 
-/* Defines for VFW missing from MinGW.
- * Remove this when MinGW incorporates them. */
-#define HWND_MESSAGE                ((HWND)-3)
-
-#define BI_RGB                      0
-
-/* End of missing MinGW defines */
-
 struct vfw_ctx {
     const AVClass *class;
     HWND hwnd;
-- 
1.9.1

